pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -362,6 +363,36 @@ func (r *ReconcileApplicationMonitoring) reconcileBlackboxExporterConfig(cr *app
 	if r.extraParams == nil {
 		r.extraParams = map[string]string{}
 	}
+
+	// Add bearer_token to ConfigMap to allow Blackbox exporter requests to get through proxy in front of Grafana service
+	blackboxServiceAccount := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "blackbox-exporter-service-account",
+			Namespace: "redhat-rhoam-middleware-monitoring-operator",
```

Comment:
Namespace shouldn't be hard-coded. It should create it in a namespace where ApplicationMonitoring CR is created.

Reply:
During creation you already do that with `cr.GetNamespace()` :)

Reply:
Fixed